packages/cli/package.json

```diff
@@ -39,7 +39,7 @@
     "colors": "1.4.0"
   },
   "dependencies": {
-    "wraplib": "0.1.0",
+    "wraplib": "0.1.1",
```

Comment:
Why are we using this library? Can't we just move utility defined in the library to the appropriate package in toolchain Ex: UriResolver to uri-resolver-js

Reply:
If we really want to create a new utility package then we should have it limited to a scope and have better naming than a generic wraplib name.

Reply:
When I put the `invokeTimeout` function in the CLI lib, I think it gets cut off by build optimizations. The default deployment modules can't find it. It works to have it duplicated in each of the deployment modules, but I thought it made sense to put it somewhere else where it can be easily tested and maintained. It doesn't naturally fit in any of the existing packages.

I created the wraplib library for experimentation, to play around with new ideas. I'm not planning to change the name or scope of it soon. I'll refactor it as it grows.

Reply:
I added copies of `invokeWithTimeout.ts` to each of the default ens deployment modules and removed `wraplib` as a dependency.

Reply:
Oh, I see now what you meant. It's weird!

Reply:
I honestly don't know what would be the best solution in this case. I think instead of patching around the actual problem `When I put the invokeTimeout function in the CLI lib, I think it gets cut off by build optimizations. The default deployment modules can't find it.` We should investigate it. maybe @namesty can help?